src/particles/plasma/PlasmaParticleContainerInit.cpp

```diff
@@ -221,12 +220,11 @@ InitParticles (const amrex::IntVect& a_num_particles_per_cell,
                 amrex::Real u[3] = {0.,0.,0.};
                 ParticleUtil::get_gaussian_random_momentum(u, a_u_mean, a_u_std, engine);
 
-                ParticleType& p = pstruct[pidx];
-                p.id()   = pid + int(pidx);
-                p.cpu()  = 0; // level 0
-                p.pos(0) = x;
-                p.pos(1) = y;
-                p.pos(2) = z;
+                int_arrdata[PlasmaIdx::id       ][pidx] = pid + int(pidx);
+                int_arrdata[PlasmaIdx::cpu      ][pidx] = 0; // level 0
```

Comment:
Likewise, for those below
```suggestion
                int_arrdata[PlasmaIdx::id ][pidx] = pid + int(pidx);
                int_arrdata[PlasmaIdx::cpu][pidx] = 0; // level 0
```